packages/landing/index.html

```diff
@@ -51,7 +52,7 @@ <h1 class="font-bold ms-2 hidden sm:text-lg md:text-2xl sm:block">
     </nav>
     <main class="flex-grow overflow-auto">
       <section id="hero" class="relative">
-        <img src="./assets/hero.png" class="w-full" />
+        <img src="./assets/hero.png" class="w-full min-h-80" />
```

Comment:
Before

![image](https://github.com/globalbibletools/gbt/assets/17209828/4215439f-926c-41c9-a514-34a585eb9df1)


After 

![image](https://github.com/globalbibletools/gbt/assets/17209828/49b43e48-c79b-43d2-bc35-00853d1c4953)


This means the image gets slightly stretched vertically on small screens, but I think it it worth it for the text spacing.

Reply:
What if instead of compressing the image, we shrink the spacing between the text and the button so it fits better over the image? We can `mb-8 sm:mb-16` on the paragraph.

Reply:
Done